packages/apps/test/commands/domains/index.test.ts

```diff
@@ -6,89 +6,88 @@ describe('domains', () => {
     acm_status_reason: 'Failing CCA check',
     app: {
       name: 'myapp',
-      id: '01234567-89ab-cdef-0123-456789abcdef'
+      id: '01234567-89ab-cdef-0123-456789abcdef',
     },
     cname: 'bar.herokudns.com',
     created_at: '2012-01-01T12:00:00Z',
     hostname: 'www.example.com',
     id: '11234567-89ab-cdef-0123-456789abcdef',
     kind: 'custom',
     updated_at: '2012-01-01T12:00:00Z',
-    status: 'succeeded'
+    status: 'succeeded',
   }, {
     acm_status: 'failing',
     acm_status_reason: 'Failing CCA check',
     app: {
       name: 'myapp',
-      id: '01234567-89ab-cdef-0123-456789abcdef'
+      id: '01234567-89ab-cdef-0123-456789abcdef',
     },
     cname: 'buzz.herokudns.com',
     created_at: '2012-01-01T12:00:00Z',
     hostname: '*.example.com',
     id: '12234567-89ab-cdef-0123-456789abcdef',
     kind: 'custom',
     updated_at: '2012-01-01T12:00:00Z',
-    status: 'succeeded'
-  }
-]
+    status: 'succeeded',
+  }]
 
   test
-    .nock('https://api.heroku.com', (api: any) => api
-      .get('/apps/myapp/domains')
-      .reply(200, herokuOnlyDomainsResponse)
-    )
-    .stdout()
-    .command(['domains', '--app', 'myapp'])
-    .it('does not show the custom domain header if there are no custom domains', ctx => {
-      expect(ctx.stdout).to.contain('=== myapp Heroku Domain\nmyapp.herokuapp.com')
-      expect(ctx.stdout).to.contain('myapp.herokuapp.com')
-      expect(ctx.stdout).to.not.contain('=== myapp Custom Domains')
-    })
+  .nock('https://api.heroku.com', api => api
+  .get('/apps/myapp/domains')
+  .reply(200, herokuOnlyDomainsResponse),
+  )
+  .stdout()
+  .command(['domains', '--app', 'myapp'])
+  .it('does not show the custom domain header if there are no custom domains', ctx => {
+    expect(ctx.stdout).to.contain('=== myapp Heroku Domain\nmyapp.herokuapp.com')
+    expect(ctx.stdout).to.contain('myapp.herokuapp.com')
+    expect(ctx.stdout).to.not.contain('=== myapp Custom Domains')
+  })
 
   test
-    .nock('https://api.heroku.com', (api: any) => api
-      .get('/apps/myapp/domains')
-      .reply(200, herokuAndCustomDomainsResponse)
-    )
-    .stdout()
-    .command(['domains', '--app', 'myapp'])
-    .it('shows a list of domains and their DNS targets when there are custom domains', ctx => {
-      expect(ctx.stdout).to.contain('=== myapp Heroku Domain\nmyapp.herokuapp.com')
-      expect(ctx.stdout).to.contain('myapp.herokuapp.com')
-      expect(ctx.stdout).to.contain('=== myapp Custom Domains')
-      expect(ctx.stdout).to.contain('Domain Name     DNS Record Type DNS Target')
-      expect(ctx.stdout).to.contain('example.com     ALIAS or ANAME  foo.herokudns.com')
-      expect(ctx.stdout).to.contain('www.example.com CNAME           bar.herokudns.com')
-      expect(ctx.stdout).to.contain('*.example.com   CNAME           buzz.herokudns.com')
-    })
+  .nock('https://api.heroku.com', api => api
+  .get('/apps/myapp/domains')
+  .reply(200, herokuAndCustomDomainsResponse),
```

Comment:
I don't think the comma dangling is working properly here with the Nock API. Is this legit javascript?

Reply:
The comma here doesn't break anything, but I do agree that it seems like eslint is getting confused here because this doesn't seem like a fitting place for a trailing comma